package.json

```diff
@@ -45,5 +45,13 @@
     "stylelint": "^13.9.0",
     "stylelint-config-recommended": "^3.0.0",
     "stylelint-config-styled-components": "^0.1.1"
+  },
+  "dependencies": {
+    "@polkadot/api": "^4.0.3",
+    "bs58": "^4.0.1",
+    "eventemitter3": "^4.0.7",
+    "lodash": "^4.17.21",
+    "query-string": "^6.14.1",
+    "store": "^2.0.12"
```

Comment:
Not sure about the addition of any of these top-level.

Reply:
the `store` is a library used to save contact data in ui-extension. Let's decide where to store that data.